core/src/main/java/greencity/constant/SwaggerExampleModel.java

```diff
@@ -15,18 +15,13 @@ public final class SwaggerExampleModel {
         + "\t<pre>\n";
 
     private static final String EXAMPLE =
```

Comment:
In this case it's acceptable, because we also have task for UpdateEcoNewsDtoRequest class, but in general case it may cause problems, becuase EXAMPLE constant is also used as part of UPDATE_ECO_NEWS constant, that has different structure.